telegram/_bot.py

```diff
@@ -5838,12 +6076,104 @@ async def delete_sticker_from_set(
         )
         return result
 
+    @_log
+    async def delete_sticker_set(
+        self,
+        name: str,
+        *,
+        read_timeout: ODVInput[float] = DEFAULT_NONE,
+        write_timeout: ODVInput[float] = DEFAULT_NONE,
+        connect_timeout: ODVInput[float] = DEFAULT_NONE,
+        pool_timeout: ODVInput[float] = DEFAULT_NONE,
+        api_kwargs: JSONDict = None,
+    ) -> bool:
+        """
+        Use this method to delete a sticker set that was created by the bot.
+
+        .. versionadded:: NEXT.VERSION
+
+        Args:
+            name (:obj:`str`): Sticker set name.
+
+        Returns:
+            :obj:`bool`: On success, :obj:`True` is returned.
+
+        Raises:
+            :class:`telegram.error.TelegramError`
+
+        """
+        data: JSONDict = {"name": name}
+        result = await self._post(
+            "deleteStickerSet",
+            data,
+            read_timeout=read_timeout,
+            write_timeout=write_timeout,
+            connect_timeout=connect_timeout,
+            pool_timeout=pool_timeout,
+            api_kwargs=api_kwargs,
+        )
+        return result
+
+    @_log
+    async def set_sticker_set_thumbnail(
+        self,
+        name: str,
+        user_id: Union[str, int],
+        thumbnail: FileInput = None,
+        # Deprecated param `thumb` not included. Would be strange to allow someone to call
+        # the new method with an old param.
```

Comment:
it never existed, so I doubt that it would even work ;)
```suggestion
```

Reply:
Right :) removing